src/ol/control/scaleline.js

```diff
@@ -174,6 +174,9 @@ ol.control.ScaleLine.prototype.updateElement_ = function() {
     ol.proj.Units.METERS;
   var pointResolution =
       ol.proj.getPointResolution(projection, viewState.resolution, center, pointResolutionUnits);
+  if (units != ol.control.ScaleLineUnits.DEGREES) {
+    pointResolution *= projection.getMetersPerUnit();
+  }
```

Comment:
can we do an if else here to avoid calling ```ol.proj.getPointResolution``` when it's not needed?

Reply:
`getPointResolution` is always needed. We're just multiplying it with the meters per unit, because these are not taken into account by `getPointResolution`.

Reply:
ah I missed the ```*=``` part, looks good then